eras/conway/impl/CHANGELOG.md

```diff
@@ -1,11 +1,14 @@
 # Version history for `cardano-ledger-conway`
 
-## 1.7.0.0
+## 1.8.0.0
 
 * Make `DELEG`, `POOL` and `GOVCERT` conform to spec-v0.8 #3628
```

Comment:
These changes were merged before the release, so wouldn't they be in 1.7.0.0 ?

Reply:
They were merged before the release happened, but that doesn't mean they were included in the release :wink: 

In other words I prepared most of the release before that PR was merged, so it didn't make it in 1.7.0.0. That's why I needed to adjust the changelog

Reply:
But the [release commit](https://github.com/input-output-hk/cardano-haskell-packages/commit/3ffcc5a9f9520d9b0d6b95fbab485f62fc011367#diff-6c78dd71a5f166dde88662222ac941c14b1797d2cc69f8b4bb18fea09a5135b5) is `bbdfc3fdb6487653f23385c3df1c9c575bf567cf`, which is exactly the one introducing these changes, no? 

https://github.com/input-output-hk/cardano-ledger/commits/bbdfc3fdb6487653f23385c3df1c9c575bf567cf

Reply:
You are totally right. For some reason I was sure that I didn't include the latest PR into the release.

Fixed.